### PR TITLE
[Feat]:Snackbar & rating components added

### PR DESCRIPTION
**Pull request by t007rushi**
🟡Snackbar added:
Used only vanilla CSS, NO JS
shows gallery snackbar 

🟡Rating added:
5-star Rating with hover effect

### CODE REVIEW

**Review by saketkum11 — APPROVED** [review]

File: components/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
+body{
+    background-color : grey;
+}
+
+.parent{
+    margin: 100px;
+    position : relative;
```
Comment (mariyasada) [inline components/snackbar.css]: use rem instead of px

File: components/snackbar.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    justify-content: space-around;
+    align-items: center;
+    width : 700px;
+    height: 3rem ;
+    margin-left : 80px;
+    border-radius : 5px;
+    background-color : rgb(51,51,51);
+    position : absolute;
+    bottom :15px;
+    left : 10px;
```
Comment (mariyasada) [inline components/snackbar.css]: use rem instead of px

File: components/snackbar.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    color : white;
+  
+    font-size : 20px;
+}
+
+.retry{
+    color: #BB86EF;
+    padding:5px;
+}
```
Comment (mariyasada) [inline components/snackbar.css]: use color varaiables

**Review by mariyasada — APPROVED** [review]
good to merge

**Comment by t007rushi** [comment]
> good to merge

Thank You for pointing out great help as pushed wrong changes

**Pull request merged**